src/storage/store.js

```diff
@@ -253,18 +265,24 @@ export default class Store extends EventTarget {
       Cookies.remove(OAUTH_FLOW_CREDENTIALS_KEY);
     }
 
-    this._signOutOnExpiredAuthToken();
+    const signOutOnInvalidAuthToken = (() => {
+      let previousToken;
+      return () => {
+        const currentToken = this.state.credentials.token;
+        const tokenDidChange = currentToken !== previousToken;
```

Comment:
I think the expression reads just as clearly as the variable and it is only used once, so I would inline it.